apps/dashboard/test/unit/rclone_util_test.rb

```diff
@@ -0,0 +1,16 @@
+require "rclone_helper"
+require 'rclone_util'
+
```

Comment:
The tests failed locally for me without `require 'test_helper'`.

Reply:
We may need to place this file somewhere else. It is telling that this failed locally when I explicitly set that path to the file but no failures happened in the PR when the tests ran. This makes me think that `unit` location is not getting picked up by the `rails` conventions. I'm going to test more to make sure this is the case, but we may need to move this file to a different location or write a `rake` task to handle it.

Reply:
`require 'test_helper'` does need to be there. Having it in the `unit` location should be fine I think, though if there's a better place for it I'm happy to change it. 
Tests were OK in the PR since if any other test runs first, `test_helper` is required already, but not if this test is the first test that runs.